### ISSUE TITLE
one-line + check-open-brace rule

### ISSUE THREAD

**Issue by fondasimone** [palantir/tslint#211]
These two blocks:

```ts
public ok(): Boolean
{
return true;
}
```

```ts
public notOk(): any
{
return true;
}
```

.. give different outcomes. The first gives a "misplaced opening brace" (as expected) the second does not. If i remove the ": any" return type, everything is fine.

**Comment by leeavital** [comment]
````typescript
class A {

    public ok(): Boolean
    {
        return true;
    }
}
````

and 

````typescript
class A {

    public ok(): any
    {
        return true;
    }

}
````

Are both producing `test.ts[4, 5]: misplaced opening brace` on the most up-to-date version of tslint. Can we close this one?

**Comment by adidahiya** [comment]
Closing this for now. Please comment if this issue persists.

**Comment by MaxKlaxxMiner** [comment]
```typescript
class A
{

    public ok(): boolean
    {
        return true;
    }

}
```

my autoformat vs-settings put all braces in new lines (valid code)

same TsLint: ```misplaced opening brace```
